remoteworkitem/github_test.go

```diff
@@ -0,0 +1,35 @@
+package remoteworkitem
+
+import (
+	"testing"
+
+	"github.com/google/go-github/github"
+)
+
+type FakeGithubIssueFetcher struct{}
+
+// ListIssues list all issues
+func (f *FakeGithubIssueFetcher) ListIssues(query string, opts *github.SearchOptions) (*github.IssuesSearchResult, *github.Response, error) {
+	if opts.ListOptions.Page == 0 {
+		one := 1
+		i := github.Issue{ID: &one}
+		isr := &github.IssuesSearchResult{Issues: []github.Issue{i}}
```

Comment:
While this is ok for demonstrating the principle, a real mock would have be more complicated than this to be able to exercise the code under test.